banyan/src/util.rs

```diff
@@ -80,6 +80,7 @@ where
     Self: 'static + Sized + Send,
     Self::Item: Send,
 {
+    #[allow(clippy::clippy::wrong_self_convention)]
     fn to_stream(self, buffer_size: usize, thread_pool: ThreadPool) -> mpsc::Receiver<Self::Item> {
```

Comment:
So clippy wants us to call this thing into_stream?

Reply:
Supposedly, yes. Shall I change it?

Reply:
if it is internal, why not.